server/utils/utils.ts

```diff
@@ -303,8 +303,8 @@ export const existsInStringArray = (key: string, arr: string[]): boolean => {
 
 export const getAttendanceSummary = (attendance: Attendance[]) => {
   const attendanceCount = attendance.length
-  const attended = attendance.filter(a => a.status === 'COMPLETED' && a.attendanceReason.code === 'ATTENDED').length
-  const notAttended = attendance.filter(a => a.status === 'COMPLETED' && a.attendanceReason.code !== 'ATTENDED').length
+  const attended = attendance.filter(a => a.status === 'COMPLETED' && a.attendanceReason?.code === 'ATTENDED').length
+  const notAttended = attendance.filter(a => a.status === 'COMPLETED' && a.attendanceReason?.code !== 'ATTENDED').length
```

Comment:
Not related to this change, but fixes a very unlikely bug I found while testing.